.sdkmanrc

```diff
@@ -2,4 +2,10 @@
 # Add key=value pairs of SDKs to use below
 
 # Java
-java=11.0.25-tem
+java=11.0.26-tem
+
+# Gradle
+gradle=8.9
+
+# Maven
```

Comment:
why set mvn here? jbang itself doesn't require/use it?

Reply:
And really there's no need for Gradle either because we're using the Gradle wrapper (which is recommended over using your own)

Reply:
ah its for easier testing of jbang export gradle/mvn

...but that actually is a good point- maybe the exports should add mvn/gradle wrapper instead?

Reply:
> .but that actually is a good point- maybe the exports should add mvn/gradle wrapper instead?

Agree.  That would solve the problem. The wrappers are missing at the moment.

Reply:
In anticipation of wrappers being generated in the future for Maven and Gradle, I have removed the entries for them from .sdkmanrc